corehq/apps/app_manager/xform.py

```diff
@@ -428,7 +429,7 @@ def add_setvalue_or_bind(ref, value):
         )
 
         if not autoset_owner_id:
-            owner_id_node.text = '-'
+            owner_id_node.text = UNOWNED_EXTENSION_OWNER_ID
```

Comment:
This owner_id looks like it is for all creates, not just extension cases. 

I tried to trace where this parameter is passed in and it looks like this happens when a case is an extension or when the owner_id is already being set by somewhere else in the case actions. 

I don't have a great, clean suggestion for this though. Maybe we could just document the cases where this is hit? Mentioning this because when I come back to this in a few months I would definitely read this as "why did they use `autoset_owner_id` if it was only for extension cases?" and be confused

Reply:
Would renaming the constant `UNOWNED_OWNER_ID` be accurate?

Reply:
all of the current references look like they are used in extension cases. I don't know if there's a separate standard for an "unowned" owner id for non-extension cases.

So with that said I'm hesitant to remove "extension" from the constant name without knowing more about the history of our "unowned" owner_id assumptions. I do think that the update as it is now is better than what it was before though (with random magic '-' laying around)